docs/writing_tests.md

```diff
@@ -18,13 +18,33 @@ can import directly from `ocsci.testlib` module with this statement:
 `from ocsci.testlib import tier1` for example.
 
 
-## Base test calsses for teams
+## Base test classes for teams
 
 Those are located in [testlib.py](../ocsci/testlib.py) which you can also
 import from `ocsci.testlib` module with statement:
 `from ocsci.testlib import manage` which is base test class for manage team.
 
 
+## Constants and Defaults
```

Comment:
@clacroix12 it will be great if you can send some update to ocs-qe about this change

Reply:
@RazTamir will do